src/main/java/longah/LongAh.java

```diff
@@ -56,30 +56,39 @@ public static void main(String[] args) {
                 case "add":
                     transactions.add(parts[1], members);
                     break;
-                case "list":
+                case "listdebts":
                     app.listAllDebts();
                     break;
-                // case "delete":
-                // if (parts.length == 2) {
-                // int index = Integer.parseInt(parts[1]);
-                // app.deleteDebt(index);
-                // } else {
-                // System.out.println("Invalid command format. Use 'delete
-                // INDEX'");
-                // }
-                // break;
-                // case "find":
-                // if (parts.length == 2) {
-                // String person = parts[1];
-                // app.findDebts(person);
-                // } else {
-                // System.out.println("Invalid command format. Use 'find
-                // PERSON'");
-                // }
-                // break;
-                // case "clear":
-                // app.clearAllDebts();
-                // break;
+                case "listtransactions":
+                    System.out.println(transactions.listTransactions());
+                    break;
+                case "delete":
+                    if (parts.length == 2) {
+                        int index = Integer.parseInt(parts[1]) - 1;
+                        transactions.remove(index);
+                    } else {
+                        System.out.println("Invalid command format. Use 'delete INDEX'");
```

Comment:
Use exceptions for these

Reply:
Would recommend to put on hold at this stage. It will be easier to move on to this stage after user i/o as well as exceptions pertaining to them is implemented